.github/workflows/test-build-spack.yaml

```diff
@@ -0,0 +1,80 @@
+name: Build ESMF Using Spack
+
+on:
+  workflow_dispatch:
+  push:
+  schedule:
+    - cron: '0 0 * * MON'
+    - cron: '0 0 * * FRI'
```

Comment:
This should be adjusted to run automatically once a day, at around midnight Mountain time on the default `develop` branch. This is part of the ESMF nightly regression testing of `develop`.